ingestion/src/metadata/ingestion/source/snowflake.py

```diff
@@ -96,6 +130,83 @@ def create(cls, config_dict, metadata_config: OpenMetadataConnection):
             )
         return cls(config, metadata_config)
 
+    def next_record(self) -> Iterable[Entity]:
+        for inspector in self.get_databases():
+            yield from self.fetch_tables(inspector=inspector, schema="")
+
+    def create_tags(self, schema: str, table_name: str, table_entity):
+        tag_category_list = self.fetch_tags(schema=schema, table=table_name)
+        table_entity.tags = []
+        for tags in tag_category_list:
+            yield tags
```

Comment:
this function just adds tags to the Table Entity right? I don't follow what we want with the `yield` here

Reply:
I've removed this